enforcer-rules/pom.xml

```diff
@@ -105,7 +105,6 @@
     <dependency>
       <groupId>org.apache.maven.resolver</groupId>
       <artifactId>maven-resolver-api</artifactId>
-      <scope>provided</scope>
```

Comment:
Given maven-resolver-api was at the end of the class path, I thought controlling the order of the dependency would solve the issue.

It did not help. https://gist.github.com/suztomo/d35ffca4e31387591a3f7b18b7366a2b